packages/client/ui/react-ui/tsup.config.ts

```diff
@@ -4,7 +4,7 @@ import { treeShakableConfig } from "../../../../tsup.config.base";
 
 const config: Options = {
     ...treeShakableConfig,
-    external: ["react", "react-dom"],
+    external: ["react", "react-dom", "@tanstack/react-query"],
```

Comment:
adding under the assumption that this will be bundled into client apps
@AlbertoElias

Reply:
have you tested it in smarter wallets? would it be too hard to remove react-query from there and see how it's being added?

Reply:
Yes, i'm testing using the quickstart in a separate project on my machine using  [yalc](https://www.npmjs.com/package/yalc)